Solutions/AristaAwakeSecurity/Workbooks/Images/Logos/AristaAwakeSecurity.svg

```diff
@@ -0,0 +1,758 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Size should be less than 5 kb, id should be guid and xlinks should not be there. Please refer [guidelines](https://microsoft.sharepoint.com/:x:/r/teams/AzureSecurityInsights/_layouts/15/guestaccess.aspx?e=aETsaQ&share=EWs_HuWia89Ns0pc1jXRWQkB7aNXX6Zi1REQcKtH4gQp5Q&isSPOFile=1)

Reply:
This has been fixed